app/src/main/java/ch/epfl/sdp/blindly/match/UserListFilter.kt

```diff
@@ -43,10 +43,10 @@ class UserListFilter {
 
         return otherUsers.filter { user ->
             if (user.showMe == EVERYONE) {
-                currentUserAge in user.ageRange?.get(0)!!..user.ageRange[1] &&
+                currentUserAge in user.ageRange?.get(0)!!..user.ageRange!![1] &&
```

Comment:
It's probably me who wrote this (my bad), but I don't understand why the `?.get(0)` is used for the first part, and `!![1]` for the second one. It could be better if both were written the same way, if possible.

Reply:
Apparently they are necessary since the attributs of the User are now vars,  Android Studio says they could've changed from the time you checked if they were not null to the time they are used